MiniApp/Classes/Extensions/MiniApp+FileManager.swift

```diff
@@ -35,8 +35,14 @@ extension FileManager {
     class func getMiniAppVersionDirectory(usingAppId id: String) -> URL? {
         do {
             let miniAppDirectory = getMiniAppDirectory(with: id)
-            let versionDirectory = try FileManager.default.contentsOfDirectory(at: miniAppDirectory, includingPropertiesForKeys: [URLResourceKey.isDirectoryKey], options: [.skipsHiddenFiles])[0]
-            return versionDirectory
+            var isDir: ObjCBool = true
+            if FileManager.default.fileExists(
+                atPath: miniAppDirectory.path,
+                isDirectory: &isDir) {
```

Comment:
Why does this need to be passed as a variable instead of just directly passing `true`?

Reply:
We need to pass a [UnsafeMutablePointer](https://developer.apple.com/documentation/foundation/filemanager/1410277-fileexists) - ObjC boolean here. I wish I would know a better way to do here, but sort of followed the old way.

Reply:
yeah, unfortunately this is a C++ legacy from ObjC.